loader/wsi.c

```diff
@@ -1796,9 +1801,10 @@ VKAPI_ATTR VkResult VKAPI_CALL terminator_GetDisplayModePropertiesKHR(VkPhysical
     }
 
     if (NULL == icd_term->dispatch.GetDisplayModePropertiesKHR) {
-        loader_log(loader_inst, VULKAN_LOADER_ERROR_BIT, 0,
+        loader_log(loader_inst, VULKAN_LOADER_WARN_BIT, 0,
                    "ICD for selected physical device is not exporting vkGetDisplayModePropertiesKHR!\n");
-        abort();
+        *pPropertyCount = 0;
+        return VK_SUCCESS;
```

Comment:
Again, because we have to handle the fallback path with these extensions, I think we need to return appropriately.

Reply:
The reason why is if some physical devices support it and others don't.  THe instance doesn't know and shouldn't crash. This is because this is an "instance" extension.  So, the application can't query if the given physical device supports the extension.  So we need to return something reasonable, yet valid.